sensing/preprocessor/pointcloud/pointcloud_preprocessor/package.xml

```diff
@@ -29,11 +29,13 @@
   <depend>pcl_ros</depend>
   <depend>libopencv-dev</depend>
   <depend>libpcl-all-dev</depend>
+  <depend>autoware_utils</depend>
+  <depend>lanelet2_extension</depend>
 
   <test_depend>ament_lint_auto</test_depend>
   <test_depend>ament_cmake_cppcheck</test_depend>
```

Comment:
Need to include `ament_lint_common` like other pakcages? https://github.com/wep21/autoware.iv.universe/blob/9232f7117f8ee0c266e354ceba80feb41ab2137d/system/autoware_state_monitor/package.xml#L32